src/main/java/org/galatea/starter/domain/IexHistoricalPrices.java

```diff
@@ -1,19 +1,31 @@
 package org.galatea.starter.domain;
 
-import lombok.Builder;
-import lombok.Data;
+import lombok.*;
+import org.galatea.starter.domain.rpsy.IexHistoricalPricesKey;
 
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.IdClass;
+import javax.xml.bind.annotation.XmlRootElement;
 import java.math.BigDecimal;
 import java.util.Date;
 
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
```

Comment:
I believe I need to explicitly set this because I've declared a constructor in the class (it creates a historical price object from a key object). Without the @AllArgsConstructor, I get no valid constructor found errors.